query-node/schema.graphql

```diff
@@ -203,6 +297,24 @@ type HttpMediaLocationEntity @entity {
   happenedIn: Block!
 }
 
+type MediaType @entity {
```

Comment:
I don't think we need to populate the query node with media types or languages

Reply:
I removed the media type. Should the language be completely removed as well? Has it been removed from metadata protobuf recently? Just double-checking.

Reply:
Language stays and they will be identified via ISO_639-1 string.